packages/checkout/widgets-lib/src/widgets/bridge/components/BridgeReviewSummaryStyles.ts

```diff
@@ -41,6 +41,10 @@ export const walletLogoStyles = (walletName: WalletProviderName) => ({
   padding: walletName === WalletProviderName.PASSPORT ? 'base.spacing.x1' : '',
   backgroundColor: 'base.color.translucent.standard.200',
   borderRadius: 'base.borderRadius.x2',
+  svg: {
+    width: 'base.icon.size.400',
+    height: 'base.icon.size.400',
+  },
```

Comment:
@sharonsheah none of these styles should be required. They all come stock standard with FramedLogo.

Reply:
This actually somehow does affect and makes the icon bigger than if I don't include these styles.

Reply:
With these styles:
![image](https://github.com/immutable/ts-immutable-sdk/assets/37326128/25f8bb9f-5302-4494-b7df-052721b38cd4)

Without styles:
![image](https://github.com/immutable/ts-immutable-sdk/assets/37326128/1ca606dd-4a1a-4b83-86bf-7f82eab15008)